js/script.js

```diff
@@ -0,0 +1,71 @@
+const navbar = document.querySelector('.navbar');
+
+document.querySelector('#menu-btn').addEventListener('click', () => {
+  navbar.classList.toggle('active');
+});
+
+const famous = [
+  {
+    id: 1,
+    name: 'John Doe',
+    image: './images/celebrities/bill.jpeg',
+    firstParagraph: 'The Porsche Club of America is the largest single-marque ',
+    secondParagraph: 'The Porsche Club of America is the largest single-marque is a great automobile',
+  },
+
+  {
+    id: 2,
+    name: 'Evans Charles',
+    image: './images/celebrities/cyrus.jpeg',
+    firstParagraph: 'The Porsche Club of America is the largest single-marque ',
+    secondParagraph: 'The Porsche Club of America is the largest single-marque',
+  },
+
+  {
+    id: 3,
+    name: 'Fredrick Smith',
+    image: './images/celebrities/gwen.jpeg',
+    firstParagraph: 'The Porsche Club of America is the largest single-marque ',
+    secondParagraph: 'The Porsche Club of America is the largest single-marque is a great automobile',
+  },
+
+  {
+    id: 4,
+    name: 'Alexis Smith',
+    image: './images/celebrities/kate.jpeg',
+    firstParagraph: 'The Porsche Club of America is the largest single-marque ',
+    secondParagraph: 'The Porsche Club of America is the largest single-marque',
+  },
+
+  {
+    id: 5,
+    name: 'Alexander Foster',
+    image: './images/celebrities/levine.jpeg',
+    firstParagraph: 'The Porsche Club of America is the largest single-marque ',
+    secondParagraph: 'The Porsche Club of America is the largest single-marque is a great automobile',
+  },
+
+  {
+    id: 6,
+    name: 'John Banascoe',
+    image: './images/celebrities/paula.jpeg',
+    firstParagraph: 'The Porsche Club of America is the largest single-marque ',
+    secondParagraph: 'The Porsche Club of America is the largest single-marque ',
+  },
+
+];
+
+// eslint-disable-next-line no-plusplus
```

Comment:
- [ ] Kindly avoid disabling linters and instead learn to work around them to fix the issues and write code that follows the specified style 👍